Modules/Filtering/Smoothing/wrapping/itkSmoothingRecursiveGaussianImageFilter.wrap

```diff
@@ -1,8 +1,6 @@
 set(WRAPPER_AUTO_INCLUDE_HEADERS OFF)
 itk_wrap_include("itkSmoothingRecursiveGaussianImageFilter.h")
 
-itk_wrap_simple_class("itk::RecursiveGaussianImageFilterEnums")
```

Comment:
Restore?

Reply:
No.  The definition occurs in two files.  Seems like a copy-n-paste error.

https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/Smoothing/wrapping/itkRecursiveGaussianImageFilter.wrap#L4

https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/Smoothing/wrapping/itkSmoothingRecursiveGaussianImageFilter.wrap#L4

Reply:
Additionally, new sanity checking identified this as being double wrapped.